balta/src/test/resources/db/postgres/06_testing.pg_types.ddl

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE testing.pg_types
+(
+    id bigint NOT NULL,
+    json_type json,
+    jsonb_type jsonb,
+    array_of_json_type json[],
+    PRIMARY KEY (id)
+);
+
+ALTER TABLE IF EXISTS testing.pg_types
+    OWNER to mag_owner;
```

Comment:
GH says that most of these SQL files have no newline at the end of the file

Reply:
Will double check.